css-layout-classes/styles.css

```diff
@@ -0,0 +1,20 @@
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+img {
+  width: 100%;
+}
+
+.column-three-fourth {
+  flex-basis: 75%;
+}
```

Comment:
You're missing a few selectors here for your columns! Like `column-one-fourth` for example